css/css-text/shaping/reference/shaping-001-ref.html

```diff
@@ -2,27 +2,22 @@
 <html  lang="en" >
 <head>
 <meta charset="utf-8">
-<title>shaping: colour</title>
+<title>reference</title>
 <style type="text/css">
 @font-face {
     font-family: 'csstest_noto';
     src: url('/fonts/noto/NotoNaskhArabic-regular.woff2') format('woff2');
-    font-weight: normal;
-    font-style: normal;
     }
-.test, .ref { border: 1px solid #02D7F6;  margin: 20px;  padding: 10px; width: 3em; font-size: 120px; font-family: "csstest_noto"; }
+.text, .ref { border: 1px solid #02D7F6;  margin: 20px;  padding: 10px; width: 3em; font-size: 120px; font-family: "csstest_noto"; }
```

Comment:
Not sure why .test was changed to .text.  If the aim is to be as minimal as possible, .test could be dropped from this file altogether.

Reply:
that's an typo / accidental commit. What do you prefer:
* correct it back to test
* drop
* leave it alone, since it doesn't apply anyway

Reply:
I leave it up to you.  

Btw, I'm applying your changes to the i18n test suite version of these tests, but at the same time making them manual tests (rather than reftests) as well as adding other content to them in line with our aim to make them useful for educational material.  That means that there will be significant differences between the incidental content of the tests in each location already, so this won't make any difference.  I used to try to keep both sets of tests as identical as possible, so i could easily see differences with a diff tool, but i've realised that i need to track differences by commit logs instead. (I'm trying to keep things modular so that Fuqiao and Atsushi can fairly easily convert them to WPT tests in future.)